.travis.yml

```diff
@@ -6,30 +6,30 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
The new parameter is only needed for the nightly version. Please move this to the related section.